foundations/introduction/motivation_and_mindset.md

```diff
@@ -166,6 +166,6 @@ This section contains helpful links to related content. It isn’t required, so
 
 *   [Managing inspiration and motivation](https://markmanson.net/do-something)
 *   [Learning to code when it gets dark](https://medium.freecodecamp.org/learning-to-code-when-it-gets-dark-e485edfb58fd#.yjh0fehje)
-*   [Improve your typing skills with Keybr.com](https://www.keybr.com/). One excellent alternative is (https://www.typing.com/). Use these keyboard trainer if you find your typing speed is holding you back. We recommend spending 5 mins every day after you start your PC.
+*   [Improve your typing skills with Keybr.com](https://www.keybr.com/). One excellent alternative is [https://www.typing.com/](https://www.typing.com/). Use these keyboard trainer if you find your typing speed is holding you back. We recommend spending 5 mins every day after you start your PC.
```

Comment:
Can we change the link text to something other than the full url please.

Something like `[typing.com](https://www.typing.com/)`

Reply:
Sure, I do that now

Reply:
Can you please check it now? I format it correctly.